src/legacy/ui/ui_bundles/ui_bundles_controller.js

```diff
@@ -168,7 +170,9 @@ export class UiBundlesController {
   }
 
   getCacheDirectory(...subPath) {
-    return this.resolvePath('../.cache', this.hashBundleEntries(), ...subPath);
+    return !IS_KIBANA_DISTRIBUTABLE
+      ? this.resolvePath('../.cache', this.hashBundleEntries(), ...subPath)
+      : this.resolvePath('../../built_assets/.cache', ...subPath);
```

Comment:
if we're including this in builds we'll probably want this in the data folder.  will this write on re-optimize?

Reply:
@jbudz during the distributable the cache is read only 😄 Actually what I'm thinking is moving the `.cache` folder to the `built_assets` both on dev and prod/distributable